cypress/e2e/utils/room-utils.ts

```diff
@@ -0,0 +1,51 @@
+import Chainable = Cypress.Chainable;
+
+export default class RoomUtils {
+
+    static openCreateRoomDialog(): Chainable<JQuery<HTMLElement>> {
+        const addRoomLabel = "Ajouter un salon";
+        const newRoomLabel = "Nouveau salon";
+        cy.get(`[aria-label="${addRoomLabel}"]`).click();
+        cy.get(`.mx_ContextualMenu [aria-label="${newRoomLabel}"]`).click();
+        return cy.get(".mx_Dialog");
+    }
+
+
+    static createPrivateRoom(roomName: string) {
+        RoomUtils.openCreateRoomDialog().within(() => {
+          // Fill name
+          const nameLabel = "Nom";
+          cy.get(`[label="${nameLabel}"]`).type(roomName);
+          // Submit
+          cy.startMeasuring("from-submit-to-room");
```

Comment:
you start measuring here, but never stop, so I don't think it measures anything. (also I don't really know what this measuring does :) )
I suggest removing it from here because start and stop should be in the same function, otherwise we'll lose them.